apps/web/src/utils/isPoolTickInRange.ts

```diff
@@ -1,6 +1,7 @@
 import { Pool } from '@pancakeswap/v3-sdk'
 
-const isPoolTickInRange = (pool: Pool, tickLower: number, tickUpper: number) => {
+const isPoolTickInRange = (pool: Pool | undefined, tickLower: number, tickUpper: number) => {
+  if (typeof pool === 'undefined') return false
   const below = pool && typeof tickLower === 'number' ? pool.tickCurrent < tickLower : undefined
```

Comment:
we can remove the pool type check if we do the check in early return
```suggestion
  const below = typeof tickLower === 'number' ? pool.tickCurrent < tickLower : undefined
```